share/pdf2htmlEX.js.in

```diff
@@ -361,10 +361,13 @@ Viewer.prototype = {
     if (url) {
       this.pages_loading[idx] = true;       // set semaphore
 
-      // add a copy of the loading indicator
-      var new_loading_indicator = this.loading_indicator.cloneNode();
-      new_loading_indicator.classList.add('active');
-      cur_page_ele.appendChild(new_loading_indicator);
+      // add a copy of the loading indicator if not already present
+      var new_loading_indicator = cur_page_ele.getElementsByClassName(this.config['loading_indicator_cls'])[0];
```

Comment:
Hmm I think clearing a page sounds more dangerous, for example maybe in future you might introduce some inner wrappers within page, which will be destroyed if we just clear the content of page. What you say?